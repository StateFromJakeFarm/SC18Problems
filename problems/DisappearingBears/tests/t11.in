41 18
5 39 70 41 67 86 31 66 2 68 20 84 85 62 57 86 42 69 86 87 3 25 95 47 94 87 42 63 45 27 89 6 44 100 50 59 40 81 62 15 37
39 22 1
28 34 6
9 28 12
16 25 10
38 36 6
16 34 1
29 6 3
10 18 7
40 41 1
6 5 9
9 31 8
7 23 11
11 28 7
29 24 2
25 13 8
38 6 7
1 38 4
18 4 6
