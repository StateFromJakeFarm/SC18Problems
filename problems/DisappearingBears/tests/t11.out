13 402 1862
